theano/sandbox/multinomial.py

```diff
@@ -346,7 +377,12 @@ def c_code(self, node, name, ins, outs, sub):
 @local_optimizer([MultinomialFromUniform])
 def local_gpu_multinomial(node):
     if type(node.op) is MultinomialFromUniform:
-        p, u = node.inputs
+        p, u, n_samples = node.inputs
```

Comment:
If we unpickle an old op, there will be only 2 inputs. Make sure to don't crash here.